metadata/transportation/road_centerlines

```diff
@@ -0,0 +1,84 @@
+
+## Description
+
+This dataset includes information such as the road name, type, speed limit, and infrastructure such as bike lanes. To see specific information about the data model including field names and descriptions, data types, coded value domains, and data model updates, see the [data model spreadsheet](https://docs.google.com/spreadsheets/d/1jQ_JuRIEtzxj60F0FAGmdu5JrFpfYBbSt3YzzCjxpfI/edit#gid=811360546) and the corresponding tabs for update notes. The color-coded section titled Addressing is primarily maintained by the local stewards, where as the section titled DOT/FHWA is primarily maintained by UDOT. UGRC spatially assigns fields in the section titled Placename/Area, based on current SGID polygon boundaries.
+
+### What is the dataset?
+
+### What is the purpose of the dataset?
+
+The Utah Roads layer plays an important role in active transportation planning in Utah. Recent collaborative efforts between UGRC, [UDOT](https://www.udot.utah.gov/connect/,) and [WFRC](https://wfrc.org/) have led to significant enhancements in the bike and pedestrian data on both the roads and trails/pathways layers. You can contribute to the on-street bike data by letting us know where we have gaps or inaccurate information.
+
+<!--- Is there a contact form people should use when they want to let us know there is a gap in the data? Should we direct them to the office email address? -->
+
+This data model is meant to suggest 'best practices'. It is not the intent that all stewards will change their current data management process to match this standard, but rather that stewards will evaluate their current practices to see how well their data 'translates' to this schema for data sharing and Next Generation 911 preparedness.
+
+<!--- Is it common practice for data stewards to change their data to fit ours?  -->
+
+### What does the dataset represent?
+
+<!--- How did we decide to delineate the individual polylines? I noticed that for many roads there is more than one polyline segment for one named road. -->
+
+### How was the dataset created?
+
+The Utah Roads dataset utilizes a data model based on [Next-Generation 911 standards](https://www.911.gov/issues/ng911/) and the Federal Highway Administration's All Roads Network Of Linear-referenced Data ([ARNOLD](https://highways.dot.gov/safety/data-analysis-tools/rsdp/rsdp-tools/arnold-reference-manual)) reporting requirements for state DOTs. UGRC adopted this data model on September 13th, 2017. The statewide roads dataset is maintained by UGRC in partnership with local government, UDOT, and the 911 community.
```

Comment:
**thought**: @gregbunce do we mention it started as TIGER data?

Reply:
maybe we omit that b/c of the connotation of tiger data. but, that's a good reminder and good context for Miriam.

Reply:
I read up a little on what TIGER data was, but I'm afraid I didn't catch the connotation--what's the story there?

Reply:
oh, i don't want to taint you with my old 1990s thinking. ha. i often think of tiger road data as not my first choice.